src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -584,7 +584,7 @@ public static object ToValue(this GraphQLValue source)
                         return bigIntegerResult;
                     }
 
-                    throw new ExecutionError($"Invalid number {str.Value}");
+                    throw new InvalidOperationException($"Invalid number {str.Value}");
```

Comment:
ExecutionErrors should not be thrown while the schema is being built

Reply:
Note that this entire `ToValue` function is a duplicate, more or less, of similar code within the CoreToAstConverter.  It should probably be deduplicated, but I have not investigated it enough to know for sure.